pkg/controller/planexecution/planexecution_controller.go

```diff
@@ -621,3 +622,8 @@ func (r *ReconcilePlanExecution) Cleanup(obj runtime.Object) error {
 
 	return nil
 }
+
+func prettyPrint(i interface{}) string {
+	s, _ := json.MarshalIndent(i, "", "  ")
+	return string(s)
+}
```

Comment:
shouldn't this be in a json util package?   this is not planexec specific

Reply:
as I reason through this more...  this function literal swallows the err.   what is the output for an error?  do we not want any indication there was an error?

Reply:
That's why I haven't put it into _util_ package - I didn't want a general purpose _prettyPrint_ method for everybody (though it can be). So far, I only saw `planexecution_controller` printing gigantic yaml object in our logs.